contracts/optimistic-ethereum/libraries/trie/Lib_MerkleTrie.sol

```diff
@@ -212,7 +210,7 @@ library Lib_MerkleTrie {
         )
     {
         return keccak256(_makeLeafNode(
-            _key,
+            Lib_BytesUtils.toNibbles(_key),
```

Comment:
`_makeLeafNode` expects nibbles as an input, this function was broken without first converting to nibbles.